src/signals/incident/components/form/MapSelectors/Asset/Selector/NearbyLayer/NearbyLayer.test.tsx

```diff
@@ -15,7 +15,7 @@ import {
   get,
   mockUseMapInstance,
   useFetchResponse,
-} from 'signals/IncidentMap/components/mapTestUtils'
+} from 'signals/IncidentMap/components/__test__/utils'
```

Comment:
about __tests__ , aren't we placing tests directly under the tested files/functions?

Reply:
The tests are directly next to their components. However, this map consists of some mockdata en function that are used in some of the tests. Didn't know where to put them else..